src/styles/card.scss

```diff
@@ -0,0 +1,118 @@
+
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  padding-inline: ($card-width - $card-img-width) / 2 - 4px;
+  border: $card-border;
+  border-radius: 5px;
+  padding-top: 2em;
+  padding-bottom: 1em;
+  color: $main-accent-color;
+  transition: transform 0.3s;
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent-color;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__name {
+      color: $card-title-color;
+    }
+  }
+
+  &__img {
+    display: flex;
+    width: $card-img-width;
+    margin: auto;
+    height: 134px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    color: $secondary-text-color;
+    margin-top: 5px;
+    margin-bottom: 9px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 17px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 12px;
+    margin-top: 23px;
+    margin-bottom: 16px;
```

Comment:
it's a good practice to use one-way margins (it's easy to read and understand, to avoid margin collapse)